### PR TITLE
add properties route

### PR DESCRIPTION
**Pull request by maxgrossman**
this route returns obj w/select properties for vpromms ids

### CODE REVIEW

File: routes/properties.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+var queryWays = require('../services/query-ways');
+var flattenDeep = require('lodash.flattendeep');
+var groupBy = require('lodash.groupby')
+var includes = require('lodash.includes');
+var assign = require('lodash.assign');
+var reduce = require('lodash.reduce');
+Promise = require('bluebird');
+module.exports = [
+  /**
+   * @api {get} /ids Return all VProMMS ids in the database
```
Comment (mileswwatkins) [inline routes/properties.js]: Can you update this documentation as well?

File: routes/properties.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+   *
+   * @apiSuccessExample {json} Success-Response:
+   *  [1, 2, 3, 4]
+   */
+  {
+    method: 'GET',
+    path: '/properties',
+    handler: function (req, res) {
+      let desired;
+      if (req.query) {
```
Comment (mileswwatkins) [inline routes/properties.js]: Then what?

File: routes/properties.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+      }
+      rp(api + 'ids')
```
Comment (mileswwatkins) [inline routes/properties.js]: Instead of make a request to the API itself, can we bring in the logic from the `/ids` route? This feels janky to me.

Comment (maxgrossman) [inline routes/properties.js]: hope for the best? sorry that's some fossilized code before I figured out the logic I needed to make this work...

**Comment by mileswwatkins** [comment]
Can you drop the dependencies on `request` and `request-promise`? Then you can self-merge this.

**Pull request merged**